python/viewer/image_viewer.py

```diff
@@ -57,12 +61,14 @@ def main():
     for radar_name in args.radar_name:
         output_path = None
         if args.output_dir is not None:
-            output_path = join(args.output_dir, radar_name+".mp4")
+            video_output_path = join(args.output_dir, radar_name+".mp4")
+            image_model_output_path = join(args.output_dir, radar_name+".pbs")
```

Comment:
we should give it a more descriptive name, such as  radar_name + "_image_models.pbs".